### DISCUSSION TITLE
WiFi NIC for Monitoring

### DISCUSSION THREAD

**Discussion by nightmare-wolf** [Security-Onion-Solutions/securityonion#4555] [General]
How can i add a Wifi NIC for monitoring?
I entered nmtui and edited my wireless interface with the wifi and credentials
I then added the wireless interface with so-monitor-add and i see the wireless interface is bonded but eth1 is in green while the wireless interface is not.
[root@donatello donatello]# nmcli con show
NAME                   UUID                                  TYPE      DEVICE
eth0                   0ceb6973-eae9-46dc-8658-546eaff06bab  ethernet  eth0
bond0                  b2f4755a-67dd-4e80-b77c-1c4df0382907  bond      bond0
bond0-slave-eth1       5a104a5d-9df5-405e-b398-15b62b198581  ethernet  eth1
bond0-slave-wlp1s27u1  07fc66ae-953a-4746-b730-b2ad079431f1  ethernet  --
GetYourOwnWifi         13b89cc8-0013-4d8e-8c25-40618570ea8b  wifi      --

The wireless interface does not show a "device" and it is showing as ethernet instead of wireless. How can i effectively add the wireless interface to monitor wireless traffic?

**Comment by weslambert** [comment]
I'd recommend tapping or mirroring the traffic before you get to the AP if you can (using the wireless interface as mgmt, and a separate interface for sniffing), as opposed to trying to use a wireless interface for sniffing.  We don't really recommend or support the use of wireless interfaces in this manner.

**Reply by nightmare-wolf** [reply]
Even though it is not supported, is there a possibility for me to do it? Possibly running a tcpdump through the wireless interface in promiscuous mode. While creating pcaps and forwarding the pcaps off to zeek for parsing?
Or installing other wifi sniffing tools that would do the wireless sniffing whilst creating the pcaps and forwarding to zeek?
